server/resources/run.py

```diff
@@ -321,3 +325,40 @@ def _fakeRun(self, rfolder: dict, token: dict) -> None:
 
     def _wait(self, secs):
         t.sleep(max(0.1, random.normalvariate(secs, math.sqrt(secs))))
+
+    @access.user
+    @autoDescribeRoute(
+        Description('Start the recorded_run job')
+        .modelParam('id', 'The ID of a run.', model=Folder, level=AccessType.WRITE,
+                    destName='run')
+        .errorResponse('Access was denied (if current user does not have write access to '
+                       'this run)', 403)
+    )
+    def startRun(self, run):
+        user = self.getCurrentUser()
+
+        runRoot = Folder().load(run['parentId'], user=user, level=AccessType.WRITE)
+        print(runRoot['meta'])
+        tale = Tale().load(runRoot['meta']['taleId'], user=user, level=AccessType.READ)
+
+        resource = {
+            'type': 'wt_recorded_run',
+            'tale_id': tale['_id'],
+            'tale_title': tale['title']
+        }
+
+        token = Token().createToken(user=user, days=0.5)
+
+        notification = init_progress(
+            resource, user, 'Recorded run',
+            'Initializing', RECORDED_RUN_STEP_TOTAL)
+
+        rrTask = recorded_run.signature(
+            args=[str(run['_id']), str(tale['_id'])],
+            girder_job_other_fields={
+                'wt_notification_id': str(notification['_id']),
+            },
+            girder_client_token=str(token['_id']),
+        ).apply_async()
+
+        return rrTask.job
```

Comment:
```suggestion
        return Job().filter(rrTask.job, user=user)
```

Reply:
Aleternatively, use `@filtermodel(model=Job)`. In any case this will require `from girder.plugins.jobs.models.job import Job`

Reply:
Done